example/thirdparty/mockito/build.sc

```diff
@@ -0,0 +1,275 @@
+import mill._, javalib._
+
+object libraries{
+
+  object versions {
+    val bytebuddy = "1.14.18"
+    val junitJupiter = "5.10.3"
+    val errorprone = "2.23.0"
+  }
+  val junit4 = ivy"junit:junit:4.13.2"
+  val junitJupiterApi = ivy"org.junit.jupiter:junit-jupiter-api:${versions.junitJupiter}"
+  val junitJupiterParams = ivy"org.junit.jupiter:junit-jupiter-params:${versions.junitJupiter}"
+  val junitPlatformLauncher = ivy"org.junit.platform:junit-platform-launcher:1.10.3"
+  val junitJupiterEngine = ivy"org.junit.jupiter:junit-jupiter-engine:${versions.junitJupiter}"
+  val junitVintageEngine = ivy"org.junit.vintage:junit-vintage-engine:${versions.junitJupiter}"
+  val assertj = ivy"org.assertj:assertj-core:3.26.3"
+  val hamcrest = ivy"org.hamcrest:hamcrest-core:2.2"
+  val opentest4j = ivy"org.opentest4j:opentest4j:1.3.0"
+
+  val bytebuddy = ivy"net.bytebuddy:byte-buddy:${versions.bytebuddy}"
+  val bytebuddyagent = ivy"net.bytebuddy:byte-buddy-agent:${versions.bytebuddy}"
+  val bytebuddyandroid = ivy"net.bytebuddy:byte-buddy-android:${versions.bytebuddy}"
+
+  val errorprone = ivy"com.google.errorprone:error_prone_core:${versions.errorprone}"
+  val errorproneTestApi = ivy"com.google.errorprone:error_prone_test_helpers:${versions.errorprone}"
+
+  val autoservice = ivy"com.google.auto.service:auto-service:1.1.1"
+
+  val objenesis = ivy"org.objenesis:objenesis:3.3"
+
+  val osgi = ivy"org.osgi:osgi.core:8.0.0"
+  val equinox = ivy"org.eclipse.platform:org.eclipse.osgi:3.20.0"
+  val bndGradle =  "biz.aQute.bnd:biz.aQute.bnd.gradle:6.4.0"
+
+  val groovy = ivy"org.codehaus.groovy:groovy:3.0.22"
+}
+
+trait MockitoModule extends MavenModule{
+  def testModuleDeps: Seq[JavaModule] = Nil
+  def testIvyDeps: T[Agg[mill.scalalib.Dep]] = Agg.empty[mill.scalalib.Dep]
```

Comment:
Can we add a `Dep` type alias to `package mill.javalib`?

Reply:
yeah we should